moved/src/move_execution/evm_native.rs

```diff
@@ -0,0 +1,862 @@
+use {
+    crate::{
+        block::HeaderForExecution,
+        primitives::{ToEthAddress, ToMoveAddress},
+    },
+    alloy::{hex::ToHexExt, primitives::map::HashMap},
+    aptos_gas_algebra::{GasExpression, GasQuantity, InternalGasUnit},
+    aptos_native_interface::{
+        safely_pop_arg, SafeNativeBuilder, SafeNativeContext, SafeNativeError, SafeNativeResult,
+    },
+    aptos_types::vm_status::StatusCode,
+    better_any::{Tid, TidAble},
+    move_binary_format::errors::PartialVMError,
+    move_core_types::{
+        account_address::AccountAddress,
+        effects::{AccountChangeSet, ChangeSet, Op},
+        ident_str,
+        identifier::{IdentStr, Identifier},
+        language_storage::StructTag,
+        resolver::MoveResolver,
+        value::{MoveStructLayout, MoveTypeLayout},
+    },
+    move_vm_runtime::{
+        native_extensions::NativeContextExtensions, native_functions::NativeFunctionTable,
+    },
+    move_vm_types::{
+        loaded_data::runtime_types::Type,
+        values::{Struct, VMValueCast, Value},
+    },
+    revm::{
+        db::{CacheDB, DatabaseCommit, DatabaseRef},
+        primitives::{
+            utilities::KECCAK_EMPTY, Account, AccountInfo, Address, BlobExcessGasAndPrice,
+            BlockEnv, Bytecode, EVMError, ExecutionResult, Log, TxEnv, TxKind, B256, U256,
+        },
+        Evm,
+    },
+    smallvec::SmallVec,
+    std::{collections::VecDeque, sync::LazyLock},
+};
+
+const EVM_NATIVE_ADDRESS: AccountAddress = AccountAddress::ONE;
+const EVM_NATIVE_MODULE: &IdentStr = ident_str!("evm");
+const ACCOUNT_STORAGE_LAYOUT: MoveTypeLayout = MoveTypeLayout::U256;
+static CODE_LAYOUT: LazyLock<MoveTypeLayout> =
+    LazyLock::new(|| MoveTypeLayout::Vector(Box::new(MoveTypeLayout::U8)));
+static ACCOUNT_INFO_LAYOUT: LazyLock<MoveTypeLayout> = LazyLock::new(|| {
```

Comment:
I assume this is a struct for `address, value, code`, but it's not obvious. We can add a comment describing this layout.

Reply:
Additional documentation added in https://github.com/MovedNetwork/op-move/pull/143/commits/4e08ce5a1f5b63e9f31ca61f93d8b9012234f49a